src/lazy/text/buffer.rs

```diff
@@ -2157,22 +2221,23 @@ mod tests {
             self
         }
 
-        fn try_match<'data, P, O>(&'data self, parser: P) -> IonParseResult<'data, usize>
+        fn try_match<'data, P, O>(&'data self, parser: P) -> IonParseResult<'data, (TextBuffer<'data>, usize)>
```

Comment:
Why was this needed?

Reply:
I changed try_match to return buffer and length so that I can just use the same method for both usecases, 1. checking length and 2. checking row-column values are right.